samples/CSharp/SourceGenerators/GeneratedDemo/GeneratedDemo.csproj

```diff
@@ -6,17 +6,25 @@
     <LangVersion>preview</LangVersion>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="People.csv" />
+  </ItemGroup>
+
   <ItemGroup>
     <AdditionalFiles Include="MainSettings.xmlsettings" />
+    <Content Include="People.csv">
```

Comment:
These should probably be input as additional files, rather than being read from the output directory. Currently if you clean and build it fails: there is essentially a deadlock where the build can't succeed because the files are missing, and the files wont be there until the build succeeds.

Reply:
Ok, make sense. BTW: I don't see the build failing but perhaps we are on a different vs version.